sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -340,6 +369,12 @@ case class FindInSet(left: Expression, right: Expression) extends BinaryExpressi
   }
 
   override def dataType: DataType = IntegerType
+
+  override def prettyName: String = "find_in_set"
+
+  override def sql: Option[String] = for {
```

Comment:
Can you see how many expressions don't support sql? It is pretty silly to duplicate so much code that does the same thing (option.map(..))